metadata/src/specs.rs

```diff
@@ -289,16 +306,23 @@ impl ConstructorSpecBuilder<state::Selector> {
 #[serde(bound = "F::TypeId: Serialize")]
 #[serde(rename_all = "camelCase")]
 pub struct MessageSpec<F: Form = MetaForm> {
-    /// The name of the message.
-    name: &'static str,
+    /// The name of the message and some optional prefixes.
+    ///
+    /// In case of trait provided messages and constructors the prefix
+    /// by convention in ink! is the name of the trait.
+    name: Vec<&'static str>,
     /// The selector hash of the message.
     #[serde(serialize_with = "serialize_as_byte_str")]
     selector: [u8; 4],
     /// If the message is allowed to mutate the contract state.
     mutates: bool,
+    /// If the message is payable by the caller.
+    #[serde(rename = "isPayable")]
```

Comment:
`[serde(rename_all = "camelCase")]` should handle this

Reply:
yeah thanks and actually I wanted to rename this to just `payable`.